java/client/src/main/java/module-info.java

```diff
@@ -9,6 +9,7 @@
     exports glide.api.models.commands.stream;
     exports glide.api.models.configuration;
     exports glide.api.models.exceptions;
+    exports glide.api.models.commands.scan;
```

Comment:
too bad spotless doesn't sort this list for us

Reply:
You can use an IntelliJ profile to auto-sort and group imports.

Reply:
Oh, maybe not for a module-info though.